src/bluehawk/actions/actions.ts

```diff
@@ -0,0 +1,3 @@
+export interface ActionArgs {
```

Comment:
Should this file be called "ActionArgs.ts" then?

Reply:
Renamed!